contracts/SmartVaultYieldManager.sol

```diff
@@ -166,7 +169,7 @@ contract SmartVaultYieldManager is ISmartVaultYieldManager, Ownable {
         IERC20(EURA).safeApprove(uniswapRouter, _balance);
         ISwapRouter(uniswapRouter).exactInput(ISwapRouter.ExactInputParams({
             path: _pathFromEURA,
-            recipient: msg.sender,
+            recipient: address(this),
             deadline: block.timestamp + 60,
```

Comment:
Do we want to have that configurable possibly (with a require min value of 60 for example). This way we can increase the deadline enforced when unforeseen issues arise, the network being congested, etc. --> it at least allows you to react if something breaks at this point.

Reply:
probably a good idea!